[doc] Add deprecation message to torch.slogdet in favor of torch.linalg.slogdet

heitorschueroff opened the pull request:
This PR is part of a larger effort to ensure torch.linalg documentation is consistent (see #50287).

Updated torch.slogdet documentation to add a deprecation message in favor of torch.linalg.slogdet.

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#51301 [doc] Add deprecation message to torch.slogdet in favor of torch.linalg.slogdet**
* #51300 [doc] Fix linalg.slogdet doc consistency issues

Differential Revision: [D26147897](https://our.internmc.facebook.com/intern/diff/D26147897)

heitorschueroff committed:
[doc] Add deprecation message to torch.slogdet in favor of torch.linalg.slogdet

[ghstack-poisoned]

mruberry commented on torch/_torch_docs.py:
```diff
@@ -9640,12 +9640,14 @@ def merge_dicts(*dicts):
 
 Calculates the sign and log absolute value of the determinant(s) of a square matrix or batches of square matrices.
 
+.. note:: :func:`torch.slogdet` is deprecated. Please use :func:`torch.linalg.slogdet`
```
"instead, ..." -> "instead."

torch.slogdet is actually just an alias for torch.linalg.slogdet now, so both are similar to NumPy's numpy.linalg.slogdet. We just want people to use the new name.

mruberry reviewed (approved):
Just one tweak.

heitorschueroff committed:
Update on "[doc] Add deprecation message to torch.slogdet in favor of torch.linalg.slogdet"

This PR is part of a larger effort to ensure torch.linalg documentation is consistent (see #50287).

Updated torch.slogdet documentation to add a deprecation message in favor of torch.linalg.slogdet.




[ghstack-poisoned]
